Content.Server/GameObjects/Components/Gravity/GravityGeneratorComponent.cs

```diff
@@ -101,6 +106,8 @@ bool IAttackHand.AttackHand(AttackHandEventArgs eventArgs)
         public bool AttackBy(AttackByEventArgs eventArgs)
         {
             if (!eventArgs.AttackWith.TryGetComponent<WelderComponent>(out var welder)) return false;
+            if (!InteractionChecks.InRangeUnobstructed(eventArgs)) return false;
```

Comment:
This check is sometimes before, sometimes after a component access check. Could it not be made more consistent? That way it could be enforced as style: "The first thing you should do in `AttackBy` is check range."

Reply:
I did this because the check method does a bit more work (raycast) and wanted to avoid that extra processing on the server side. But it's probably insignificant, can move it to the beginning for consistency.